packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec

```diff
@@ -1,35 +1,28 @@
 # template: foreman_plugin
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-%{!?_root_localstatedir:%global _root_localstatedir %{_localstatedir}}
```

Comment:
this variable is used a few times in the spec, but now that you're not doing any funny SCL things, you can just replace it with the "original" `_localstatedir`

Reply:
@sbernhard Can you have a look, please :)

Reply:
just do a `sed -i 's/_root_localstatedir/_localstatedir/g' packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec` ;)

Reply:
:woman_shrugging: